packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1834,6 +1860,14 @@ class _TransformedPointerScrollEvent extends _TransformedPointerEvent with _Copy
     super.debugFillProperties(properties);
     properties.add(DiagnosticsProperty<Offset>('scrollDelta', scrollDelta));
   }
+
+  @override
+  RespondPointerEventFn get _respond => original._respond;
+
+  @override
+  void respond({required bool allowPlatformDefault}) {
+    original.respond(allowPlatformDefault: allowPlatformDefault);
+  }
```

Comment:
Do you need this override since the superclass will just use the overridden _respond?

Reply:
Yes, because this subclass is written with `implements` instead of `extends` :)

(IMO `_TransformedPointerEvent` could be a `mixin _Transformable`, but we can shave that yak later, didn't look much into it! :P)